### DISCUSSION TITLE
Assign default values to returned data

### DISCUSSION THREAD

**Discussion by pke** [vercel/swr#537] [General] [answered]
I am sure its an ES6 feature which I just can't seem to see here, but is it possible to assign default values to returned data?

`const { data = { items = [] } } = useSWR("/order", get)`

so that I do not have to conditionally access `data.items?.length` everywhere?

**Comment by sergiodxa** [comment]
You can use the `initialValue` option for that, or rely on JS default values as in your example.

**Reply by elfico** [reply, marked as answer]
It's now `fallbackData` in the latest release
